packages/amplify-e2e-tests/src/__tests__/api_5.test.ts

```diff
@@ -123,6 +123,16 @@ describe('amplify add api (REST)', () => {
       restrictAccess: true,
       allowGuestUsers: false,
     });
+
+    // add more paths to and test policy slicing
+    for (let i = 0; i < 15; i++) {
+      await addRestApi(projRoot, {
+        isFirstRestApi: false,
+        existingLambda: true,
+        restrictAccess: true,
+        allowGuestUsers: true,
+      });
+    }
```

Comment:
@yuth I think you need to pass in a path to the e2e helper, since the default is always `/items`

Reply:
good catch

Reply:
Oh, perhaps I misread this. I thought this was adding a new path to an existing API, but this should be fine if they're all distinct resources.